internal/services/mssqlmanagedinstance/mssql_managed_instance_resource.go

```diff
@@ -159,6 +155,57 @@ func (r MsSqlManagedInstanceResource) Arguments() map[string]*pluginsdk.Schema {
 			}),
 		},
 
+		"administrator_login": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			ForceNew:     true,
+			AtLeastOneOf: []string{"administrator_login", "microsoft_entra_administrator"},
+			RequiredWith: []string{"administrator_login", "administrator_login_password"},
+			ValidateFunc: validation.StringIsNotEmpty,
+		},
+
+		"administrator_login_password": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			Sensitive:    true,
+			AtLeastOneOf: []string{"administrator_login_password", "microsoft_entra_administrator"},
+			RequiredWith: []string{"administrator_login", "administrator_login_password"},
+			ValidateFunc: validation.StringIsNotEmpty,
+		},
+
+		"microsoft_entra_administrator": {
```

Comment:
Since the rest of the provider calls this `azure_active_directory_administrator`, whilst this has been rebranded, for consistency with the rest of the provider for the moment we should continue calling this `azure_active_directory_administrator` - we'll handle renaming all of these to `microsoft_entra_administrator` at the same time (presumably in 4.0) - as such can we update this?

Reply:
Yes, the code has been updated. Could you please take another look?

Reply:
ty azure active directory also matches the API/SDK names